geode-core/src/main/java/org/apache/geode/internal/JvmSizeUtils.java

```diff
@@ -139,11 +139,7 @@ public static long roundUpSize(long size) {
     // Round up to the nearest 8 bytes. Experimentally, this
     // is what we've seen the sun 32 bit VM do with object size.
     // See https://wiki.gemstone.com/display/rusage/Per+Entry+Overhead
```

Comment:
Can you make these comments javadocs and since this wiki is defunct can you remove the link.